### ISSUE TITLE
AssertionError: "a > b" was NOT fulfilled in parse_to_unicode

### ISSUE THREAD

**Issue by MartinThoma** [py-pdf/PyPDF2#990]
When trying to extract the text from a PDF, I get an exception.

## Environment

```bash
$ python -m platform
Linux-5.4.0-113-generic-x86_64-with-glibc2.31

$ python -c "import PyPDF2;print(PyPDF2.__version__)"
2.2.0
```

## MCVE: Code + PDF

This is a minimal, complete example that shows the issue with [923767.pdf](https://corpora.tika.apache.org/base/docs/govdocs1/923/923767.pdf):

```python
from PyPDF2 import PdfReader
reader = PdfReader("923767.pdf")
reader.pages[0].extract_text()
```

gives

```
  File "/home/moose/Github/py-pdf/PyPDF2/PyPDF2/_page.py", line 1301, in extract_text
    return self._extract_text(self, self.pdf, space_width, PG.CONTENTS)
  File "/home/moose/Github/py-pdf/PyPDF2/PyPDF2/_page.py", line 1124, in _extract_text
    cmaps[f] = build_char_map(f, space_width, obj)
  File "/home/moose/Github/py-pdf/PyPDF2/PyPDF2/_cmap.py", line 21, in build_char_map
    map_dict, space_code, int_entry = parse_to_unicode(ft, space_code)
  File "/home/moose/Github/py-pdf/PyPDF2/PyPDF2/_cmap.py", line 225, in parse_to_unicode
    assert a > b
AssertionError
```

**Comment by pubpub-zz** [comment]
error in the assert; however this is extra test not improving performances :  I propose to remove it

**Comment by MartinThoma** [comment]
Closed by #995 :-)
